### DISCUSSION TITLE
十分好的APP，爱来自河北

### DISCUSSION THREAD

**Discussion by icarus667** [share121/inter-knot#5361] [灌水]
![1726892419219](https://github.com/user-attachments/assets/82444197-f154-45e6-aee2-a50c803ec883)
是什么都能发吗

**Comment by fzdqz** [comment]
16+以上内容不能发_(´_`」 ∠)_

**Comment by KELERSAMA** [comment]
可以看看
#4990   搜索词：绳网常见问题的解决方法 
#4628   搜索词：绳网社区规则详情与准则 
#1436   搜索词：如何让你的帖子更加与众不同 
#4639   搜索词：在github使用米游社表情包
#2106     顺带辟谣一下
<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by fufuTvT** [comment]
谁家小孩！<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >

**Comment by TNCNET** [comment]
并不是，遵循很多类似的公开平台的相关规定。
•禁止NSFW内容/18+/敏感内容
•别发广告，不能挂人，等等。

别的二楼有提哦

**Comment by 210flight** [comment]
这是我家小孩😋
